### DISCUSSION TITLE
Add something to the title bar of viewContainer programmatically

### DISCUSSION THREAD

**Discussion by KrabsWong** [microsoft/vscode-discussions#69] [Extension Development QnA] [answered]
I'm developing an extension to show some views with a viewContainer, I wanna show up a flag (or an icon, text, dropdown list the same as the one in "Debug view"), since the data is different with my different dev environment(such as production, staging, test).

But it seems there is no api to support it?

<img width="510" alt="image" src="https://user-images.githubusercontent.com/12368943/179525340-ce35fee9-027a-4fc4-9f0c-aed95f52dc0e.png">

**Comment by alefragnani** [comment, marked as answer]
Hi @KrabsWong ,

> But it seems there is no api to support it?

You are right, there is no API for that. And even if you use a WebView, you won't be able to create such UI.

You can customize the content inside each Panel (such as Variables, Watch, Call Stack), but not the Side Bar.

Hope this helps

**Reply by KrabsWong** [reply]
Many thanks for your response.

I've changed my solution to use status bar showing up the environment info.
